src/applications/disability-benefits/all-claims/tests/config/schema.js

```diff
@@ -0,0 +1,1099 @@
+// This file to be moved to vets-json-schema repo before form hits production
```

Comment:
Is there a reason this file is in `src/applications/disability-benefits/all-claims/tests/config/schema.js`